src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
```

Comment:
Consider using a more descriptive variable name instead of 'total', such as 'totalCost' or 'rentalCost'.